src/guacscanner/guacscanner.py

```diff
@@ -51,6 +51,7 @@
 import string
 import sys
 import time
+import typing
```

Comment:
We have historically imported only the pieces of this library that we are using.

```suggestion
from typing import Optional
```

Reply:
In this case both `typing.Optional` and `schema.Optional` are floating around and I can't have them clobbering each other.

Reply:
I would prefer we switch the `schema` import to `import schema` and prefix the pieces we use and leave the `from typing import Optional`. The `typing` imports are more likely to be interspersed in the code base while the `schema` stuff will be isolated to where we do argument validation.

Reply:
Please see comment 0b35318.

Reply:
Thanks!